docs/pages/textarea.js

```diff
@@ -36,219 +53,270 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
           />
         </MainSection.Subsection>
       </MainSection>
-      <Example
-        id="basicExample"
-        name="Example"
-        description={`
-    A \`TextArea\` will expand to fill the width of the parent container.
-  `}
-        defaultCode={`
-function Example(props) {
-  const [value, setValue] = React.useState('')
-  return (
-    <TextArea
-      id="aboutme"
-      onChange={({value}) => setValue(value)}
-      placeholder="Write something about yourself..."
-      label="With a placeholder"
-      value={value}
-    />
-  );
-}
-`}
-      />
-      <Example
-        id="disabledExample"
-        name="Example: Disabled"
-        defaultCode={`
-function Example(props) {
-  const [value, setValue] = React.useState('')
-  return (
-    <TextArea
-      disabled
-      id="disabled"
-      onChange={({value}) => setValue(value)}
-      placeholder="Write something about yourself..."
-      label="With a placeholder"
-      value={value}
-    />
-  );
-}
-`}
-      />
-      <Example
-        id="helperText"
-        name="Example: Helper Text"
-        description={`Whenever you want to provide more information about a form field, you should use \`helperText\`.`}
-        defaultCode={`
-function Example(props) {
-  const [value, setValue] = React.useState('')
-  return (
-    <Box padding={2} color="white">
-      <TextArea
-        id="aboutmemore"
-        onChange={({value}) => setValue(value)}
-        placeholder="Write something about yourself..."
-        helperText="I love to sail, run and visit remote places"
-        label="With a placeholder"
-        value={value}
-      />
-    </Box>
-  );
-}
-`}
-      />
-      <Example
-        id="errorMessageExample"
-        name="Example: Error message"
-        description={`
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Basic"
```

Comment:
This title never really conveyed what was going on. The description talks about the width, so perhaps "Width" instead?